docs/README.md

```diff
@@ -39,6 +39,15 @@ Moov Customers has several models which are used throughout the HTTP endpoints.
 
 `Account` represents a demand-deposit account at a financial institution. The account number is encrypted. See the [API documentation](https://moov-io.github.io/customers/#post-/customers/{customerID}/accounts) for creating an Account.
 
+#### Account Validation
+
+In order to use a new account for ACH transactions, it will need to be validated. Moov Customers supports following strategies that can be used for account validation:
```

Comment:
```suggestion
In order to use an account for ACH transactions, it will need to be validated. This is to ensure access and authorization to the financial instrument. Customers supports following strategies that can be used for account validation:
```